acvm/src/compiler/mod.rs

```diff
@@ -26,6 +26,7 @@ pub enum CompileError {
 /// Applies [`ProofSystemCompiler`][crate::ProofSystemCompiler] specific optimizations to a [`Circuit`].
 pub fn compile(
     acir: Circuit,
+    inputs: Vec<Witness>,
```

Comment:
I think we should track this on the `Circuit` struct.

Reply:
Done